packages/access-om2-bgc/package.py

```diff
@@ -24,15 +24,7 @@ class AccessOm2Bgc(BundlePackage):
     depends_on("libaccessom2~deterministic", when="~deterministic", type="run")
     depends_on("cice5+deterministic", when="+deterministic", type="run")
     depends_on("cice5~deterministic", when="~deterministic", type="run")
-    depends_on(
-        "mom5+deterministic type=ACCESS-OM-BGC",
-        when="+deterministic",
-        type="run"
-    )
-    depends_on(
-        "mom5~deterministic type=ACCESS-OM-BGC",
-        when="~deterministic",
-        type="run"
-    )
+    depends_on("mom5+deterministic", when="+deterministic", type="run")
+    depends_on("mom5~deterministic", when="~deterministic", type="run")
```

Comment:
There's probably a nicer way to do this than my suggestion below, but this SPR should use `mom5@legacy-access-om2-bgc`.

```suggestion
    depends_on("mom5@legacy-access-om2-bgc+deterministic", when="+deterministic", type="run")
    depends_on("mom5@legacy-access-om2-bgc~deterministic", when="~deterministic", type="run")
```